Makefile

```diff
@@ -33,9 +34,20 @@ GOPROXY := https://proxy.golang.org
 endif
 export GOPROXY
 
+# Use GOPRIVATE environment variable if set
+GOPRIVATE := $(shell go env GOPRIVATE)
+export GOPRIVATE
+
 # Active module mode, as we use go modules to manage dependencies
 export GO111MODULE=on
 
+# Base docker images
+
+DOCKERFILE_CONTAINER_IMAGE ?= docker.io/docker/dockerfile:1.4
+DEPLOYMENT_BASE_IMAGE ?= gcr.io/distroless/static
+DEPLOYMENT_BASE_IMAGE_TAG ?= nonroot-${ARCH}
+BUILD_CONTAINER_ADDITIONAL_ARGS ?=
```

Comment:
Is this supposed to have something on the other side of the `?=` sign?

Reply:
Yes, container additional arguments are empty by default and using for mounting external volumes, i.e. netrc or others, if needed.